docs/book/src/quick-start.md

```diff
@@ -25,19 +25,29 @@ arch=$(go env GOARCH)
 
 # download kubebuilder and extract it to tmp
 curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
+```
+
+You must configure the Kubernetes binaries required for the [envtest][envtest], run:  
 
+```bash
 # move to a long-term location and put it on your path
 # (you'll need to set the KUBEBUILDER_ASSETS env var if you put it somewhere else)
 sudo mv /tmp/kubebuilder_2.3.1_${os}_${arch} /usr/local/kubebuilder
 export PATH=$PATH:/usr/local/kubebuilder/bin
 ```
 
+<aside class="note">
+<h1>Kubernetes binaries</h1>
+
+The above setup is no longer required if you are using plugin version `v3+. 
```

Comment:
With the comment above, we could probably delete this new section.

Reply:
I am not sure. I think is better we keep until the v3 alpha plugin version is stable and default at least. Then, we will be able to deprecate the v2 and remove the note as the text above you must run. See that currently, users must run.